SPECS/grub2/grub2.spec

```diff
@@ -266,9 +266,10 @@ popd
 #make sure all the files are same between two configure except the /usr/lib/grub
 %check
 %ifarch x86_64
-diff -sr install-for-efi/sbin install-for-pc/sbin && \
-diff -sr install-for-efi%{_bindir} install-for-pc%{_bindir} && \
-diff -sr install-for-efi%{_sysconfdir} install-for-pc%{_sysconfdir} && \
+# Note: bin+sbin binaries are expected to differ due to different CFLAGS
+#diff -sr install-for-efi/sbin install-for-pc/sbin
```

Comment:
Can we remove these two commented-out lines?

Reply:
sure - removed